src/Explorer/Explorer.ts

```diff
@@ -1953,6 +1956,16 @@ export default class Explorer {
     return Q();
   }
 
+  public setFeatureFlagsFromFlights(flights: readonly string[]): void {
+    if (!flights) {
+      return;
+    }
+
+    if (flights.filter((flightName: string) => flightName === Constants.Flights.settingsV2).length === 1) {

```

Comment:
>filter [](start = 16, length = 6)

Can we simply use indexof? filter creates new array which seems unnecessary here.